rust/src/ldap/ldap.rs

```diff
@@ -426,4 +426,24 @@ pub unsafe extern "C" fn rs_ldap_register_parser() {
     } else {
         SCLogDebug!("Protocol detection and parser disabled for LDAP.");
     }
+
+    parser.ipproto = IPPROTO_UDP;
```

Comment:
Should not there be an update to schema.json with `ldap_udp` somewhere ?

Reply:
Right, I edited the file but forgot to include it in the commit.